packages/core/src/Stores/client-store.js

```diff
@@ -1567,7 +1569,7 @@ export default class ClientStore extends BaseStore {
                 this.root_store.ui.toggleResetEmailModal(true);
             }
         }
-        const client = this.accounts[this.loginid];
+        const client = this.accounts[this.loginid] || { token: localStorage.getItem('config.account1') };
```

Comment:
Every new changes in client-store is for backward compatibility.

Reply:
[Q] what happen if localStorage.getItem is returning empty, or undefined, is it fine to have {token: undefined}?